### PR TITLE
Delaware tour mini 371 change json

### PR DESCRIPTION
**Pull request by abriggs-usgs**
Before making a pull request
----------------------------
First . . .
- [x] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [x] Make sure all tests run

Then check for accessibly compliance
- [] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [x] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [x] Update the changelog appropriately

Change to Use One 'Master' Monitoring Location Data File
-----------
Switched to use a single monitoring location JSON file and update the layers programmatic-ly rather than many files added manually. 

Description
-----------
Changes - summary
Use a single master monitoring location file
loads layer dynamically
delete all other json files
have a master file that it is a template (it is complete except for the 'associatedRegions', the 'locationFeatures' should be correct
changed the code that formerly referenced an object to reference an array
fixed a number of minor issues related to the JSON change
adjusted the 'mapStory' to use the new layers
created a master style sheet for map related colors, dot sizes, etc.
used the new master style sheet

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

**Review by mwernimont — APPROVED** [review]

File: src/assets/mapStory/mapStory.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ export default {
             "title": "Next Generation Observing Systems",
             "content": "Emergency managers and water resource managers rely on the USGS stream gage network and the associated water delivery to provide data to address complex challenges involving water. This network is functioning but was designed many years ago and needs major upgrades to meet increasingly complex water challenges facing the Nation.",
             "tourType": "none",
-            "layersToHide": ["all_locations", "monitoring-location-clusters", "monitoring-location-cluster-count",  "monitoring-location-unclustered-point", "physiographic_provinces"],
-            "hiddenLayersToShow": [],
+            "layersToHide": ["monitoring-location-clusters", "monitoring-location-cluster-count",  "monitoring-location-unclustered-point", "physiographic_provinces"],
+            "hiddenLayersToShow": ["new_gage_delaware_basin", "enhanced_gag_delaware_basin", "camera_delaware_basin", "specific_conductance_delaware_basin", "temperature_delaware_basin"],
```
Comment (mhines-usgs) [inline src/assets/mapStory/mapStory.js]: typo gag/gage

File: src/assets/mapStory/mapStory.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ export default {
...
             "tourType": "none",
             "layersToHide": ["all_locations", "monitoring-location-clusters", "monitoring-location-cluster-count",  "monitoring-location-unclustered-point", "physiographic_provinces"],
-            "hiddenLayersToShow": [],
+            "hiddenLayersToShow": ["new_gage_delaware_basin", "enhanced_gag_delaware_basin", "camera_delaware_basin", "specific_conductance_delaware_basin", "temperature_delaware_basin"],
```
Comment (mhines-usgs) [inline src/assets/mapStory/mapStory.js]: typo gag/gage

File: src/assets/mapStory/mapStory.js [hunk 3]
```diff
@@ -50,7 +50,7 @@ export default {
             "content": "The USGS collects water related data from more than 10,000 gages in the basin.",
             "tourType": "none",
             "layersToHide": ["all_locations","physiographic_provinces"],
-            "hiddenLayersToShow": [],
+            "hiddenLayersToShow": ["new_gage_delaware_basin", "enhanced_gag_delaware_basin", "camera_delaware_basin", "specific_conductance_delaware_basin", "temperature_delaware_basin"],
```
Comment (mhines-usgs) [inline src/assets/mapStory/mapStory.js]: typo gag/gage

File: src/assets/mapStory/mapStory.js [hunk 4]
```diff
@@ -69,7 +69,7 @@ export default {
             "content": "In 2018, seventeen new monitoring systems were added to the Delaware River Basin region to address monitoring gaps to improve the quantification of the amount of water in the headwater and tributary streams and to track the salt front in the mainstem",
             "tourType": "none",
             "layersToHide": ["all_locations", "monitoring-location-clusters", "monitoring-location-cluster-count",  "monitoring-location-unclustered-point", "cameras", "specific_conductance", "enhanced_gage", "temperature", "physiographic_provinces"],
-            "hiddenLayersToShow": [],
+            "hiddenLayersToShow": ["new_gage_delaware_basin", "enhanced_gag_delaware_basin", "camera_delaware_basin", "specific_conductance_delaware_basin", "temperature_delaware_basin"],
```
Comment (mhines-usgs) [inline src/assets/mapStory/mapStory.js]: typo gag/gage

File: src/assets/mapStory/mapStory.js [hunk 5]
```diff
@@ -164,7 +164,11 @@ export default {
             "content": "The Delaware River Basin is divided up into five physiographic provinces representing distinctive areas having common topography, rock types and structure, and geologic and geomorphic history",
             "tourType": "none",
             "layersToHide": ["monitoring-location-clusters", "monitoring-location-cluster-count",  "monitoring-location-unclustered-point", "cameras", "enhanced_gage", "temperature", "new_gage", "specific_conductance","drb_boundary"],
-            "hiddenLayersToShow": ["physiographic_provinces"],
+            "hiddenLayersToShow": ["physiographic_provinces",
```
Comment (mhines-usgs) [inline src/assets/mapStory/mapStory.js:167]: typo gag/gage in each of these lines

File: src/assets/mapStory/mapStory.js [hunk 6]
```diff
@@ -183,19 +187,42 @@ export default {
...
                     "management goal. However, sea level rise may require new management measures, operations plans and/or additional water to maintain control of salinity.",
             "tourType": "all_locations",
-            "layersToHide": ["monitoring-location-clusters", "monitoring-location-cluster-count", "physiographic_provinces"],
-            "hiddenLayersToShow": ["drb_boundary"],
+            "layersToHide": ["monitoring-location-clusters", "monitoring-location-cluster-count", "cameras", "enhanced_gage", "temperature", "new_gage", "specific_conductance","drb_boundary","physiographic_provinces"],
+            "hiddenLayersToShow": ["drb_boundary", "new_gage_delaware_basin", "enhanced_gag_delaware_basin", "camera_delaware_basin", "specific_conductance_delaware_basin", "temperature_delaware_basin"],
```
Comment (mhines-usgs) [inline src/assets/mapStory/mapStory.js]: typo gag/gage

File: src/assets/mapStory/mapStory.js [hunk 7]
```diff
@@ -183,19 +187,42 @@ export default {
...
+                "zoom": 6.5,
+                "pitch": 20,
+                "speed": 0.5,
+                "essential": true
+            },
+            "title": "Sample Test Bed (not real locations)",
+            "content": "This is a place holder to serve as an example of functionality",
+            "tourType": "none",
+            "layersToHide": ["monitoring-location-clusters", "monitoring-location-cluster-count",  "monitoring-location-unclustered-point", "cameras", "enhanced_gage", "temperature", "new_gage", "specific_conductance","drb_boundary"],
+            "hiddenLayersToShow": ["new_gage_test_bed", "enhanced_gag_test_bed", "camera_test_bed", "specific_conductance_test_bed", "temperature_test_bed"],
```
Comment (mhines-usgs) [inline src/assets/mapStory/mapStory.js]: typo gag/gage

**Pull request merged**